frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/prefill-vm-template-state-update.ts

```diff
@@ -209,7 +210,7 @@ export const prefillVmTemplateUpdater = ({ id, dispatch, getState }: UpdateOptio
     const standaloneDataVolumeLookup = createBasicLookup<V1alpha1DataVolume>(dataVolumes, getName);
 
     const vmDisks = commonTemplateName
-      ? getDisks(vm).filter((d) => d.name !== 'rootdisk')
+      ? getDisks(vm).filter((d) => d.name !== VM_TEMPLATE_NAME_PARAMETER)
```

Comment:
@yaacov @metalice please note this is the correct fix, we should stick to updaters.